marinerRun.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+cleanup() {
+  killall gen3-fuse
+  fusermount -uz /$COMMONS_DATA/data
+  echo "gen3fuse exited successfully"
+  exit 0
+}
+
+# maybe use a more vertically scalable method
+# than passing manifest via environment variable
+# Q. What if the manifest is very big?
+echo "writing manifest"
+echo $GEN3FUSE_MANIFEST
+echo $GEN3FUSE_MANIFEST > /manifest.json
+
+echo "running gen3-fuse.."
+gen3-fuse \
+-config=/fuse-config.yaml \
+-manifest=/manifest.json \
+-mount-point=/$COMMONS_DATA/data \
+-hostname=https://$HOSTNAME \
+-wtsURL=http://workspace-token-service.$GEN3_NAMESPACE >/proc/1/fd/1 2>/proc/1/fd/2
+
+echo "here is the mounted directory:"
+ls -R /$COMMONS_DATA
+
+trap cleanup SIGTERM
+if [ $MARINER_COMPONENT == "engine" ]; then
+  echo "waiting for engine to finish.."
+  while [[ ! -f /$ENGINE_WORKSPACE/workflowRuns/$RUN_ID/done ]]; do
+    :
+  done
+else
+  echo "waiting for task to finish.."
+  while [[ ! -f $TOOL_WORKING_DIR\done ]]; do
```

Comment:
can you link to the code where this environment variable is defined?

Reply:
and also -- to the code where this script is used?

Reply:
Yes, here's the spot: https://github.com/uc-cdis/mariner/blob/master/mariner/k8s.go#L219-L224

Reply:
mariner config, has the command for the gen3-fuse container: https://github.com/uc-cdis/gitops-dev/blob/master/mattgarvin1.planx-pla.net/manifest.json#L256-L259

and that command gets pulled out of the config here: https://github.com/uc-cdis/mariner/blob/b20a2fba97d16d044ce276f01b9cba03e49f3b7c/mariner/k8s.go#L532-L544

as part of this function that constructs the gen3-fuse container spec within mariner: https://github.com/uc-cdis/mariner/blob/b20a2fba97d16d044ce276f01b9cba03e49f3b7c/mariner/k8s.go#L68-L74